src/components/Card/PriceDetailsCard.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
```

Comment:
Same as the above mentioned point use Absolute Import. It will make imports easy when your code grows in size.